lighthouse-core/gather/gatherers/trace-nodes.js

```diff
@@ -0,0 +1,99 @@
+/**
+ * @license Copyright 2020 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const Gatherer = require('./gatherer.js');
+const pageFunctions = require('../../lib/page-functions.js');
+const TraceProcessor = require('../../lib/tracehouse/trace-processor.js');
+
+/**
+ * @return {LH.Artifacts['TraceNodes']}
+ */
+/* istanbul ignore next */
+function collectTraceNodes() {
+  /** @type {Array<HTMLElement>} */
+  // @ts-ignore - put into scope via stringification
+  const markedElements = getElementsInDocument('[lhtemp]'); // eslint-disable-line no-undef
```

Comment:
If you make the constant I mention, you'll need to add it as a parameter to this fn too.

Reply:
Added the constant, but I can't figure out how to pass it through to collectTraceNodes() since it's called via driver.evaluateAsync().

Reply:
on L92: `return (${collectTraceNodes})(${MY_CONST});`

Reply:
Ah, I've got it now. I tried what you have above and it failed because it read the value of ${MY_CONST} as a variable. Just had to surround it with single quotes.